### ISSUE TITLE
collection: EOL of ansible-core 2.13 on November 26, 2023

### ISSUE THREAD

**Issue by berndfinger** [sap-linuxlab/community.sap_install#507]
So we have to change our `meta/runtime.yml` files to `2.14`.
At the same time, we should also change move from `ansible-lint` version `6.8.6` to the lastest version, which currently is `6.21.1`. Depending on the amount of changes required for `ansible-lint`, we can maybe do these changes in one PR.

**Comment by berndfinger** [comment]
Let's use `2.12` for the community version, to increase the amount of supported releases for control nodes and use `2.14` only in the downstream version which is uploaded to Automation Hub.

**Comment by berndfinger** [comment]
Just FYI - I have used:
```
cd (collection root)
(modified meta/runtime.yml)
find roles -name runtime.yml -exec echo cp meta/runtime.yml {} \; -print
find roles -name runtime.yml -exec cp meta/runtime.yml {} \; -print
find . -name runtime.yml -exec cat {} \; -print
```
to make the changes and verify the correct settings.

**Comment by berndfinger** [comment]
Solved with #540.
